en/components/grids_templates/keyboard_navigation.md

```diff
@@ -22,35 +22,80 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
+* @@igxName `paginator` (if enabled);
+
+>[!NOTE]
+> Due to this behavior change to navigate between the cells with `tab` and `shift + tab` key is no longer supported in the @@igxName.
+> By pressing `tab` key you are going through the following sequence: "Grid Toolbar" -> "Grid Headers" -> "Grid Body" -> "Summaries" -> "Footer / Grid
+> Paginator". Also keep in mind that if you add a focusable element in the grid template it will be added to the page tab sequence.
+
+### Header Navigation
+
+We introduced a full keyboard navigation support in the grid headers, so now is possible to navigate with the arrows keys through them. Also we provide a number of key combinations that trigger a different column functionality like filtering, sorting, grouping and etc.
+When you focus the header container of the grid and press one of the following key combinations, the described behavior is performed.
+
+#### Key Combinations
+
+ - `Arrow Up` - navigates one cell up in the headers (no wrapping); Available only when multi row layout or column groups are defined;
+ - `Arrow Down` - navigates one cell down in the headers (no wrapping); Available only when multi row layout or column groups are defined;
  - `Arrow Left` - navigates one cell left (no wrapping between lines);
  - `Arrow Right` - navigates one cell right (no wrapping between lines);
+ - `Ctrl + Arrow Left` - moves to leftmost cell in  the row; if there is MRL or MCH to the leftmost cell at the same level;
+ - `Home` - moves to leftmost cell in  the row; if there is MRL or MCH to the leftmost cell at the same level;
+ - `Ctrl + Arrow Right` - moves to rightmost cell in row; if there is MRL or MCH to the rightmost cell at the same level;
+ - `End` - moves to rightmost cell in row; if there is MRL or MCH to the rightmost cell at the same level;
+ - `Alt + L` - opens Advanced filtering dialog if advanced filtering is enabled;
+ - `Ctrl + Shift + L` - opens the Excel style filter or default (row) filter if the column is filterable;
+ - `Ctrl + Arrow Up` - sorts the active column header in ASC order. If the column is already sorted in ASC, it will remove the sorting;
+ - `Ctrl + Arrow Down` - sorts the active column header in DSC order. If the column is already sorted in DSC, it will remove the sorting;
+ - `Shift + Alt + Arrow Left` - group by the active column /if the column is marked as groupable/;
+ - `Shift + Alt + Arrow Right` - ungroup by the active column /if the column is marked as groupable/;
+ - `Alt + Arrow Left` or `Alt + Arrow Up` - collapses the column group header if the column is not already collapsed;
+ - `Alt + Arrow Right` or `Alt + Arrow Down` - expands the column group header if the column is not already expanded;
+ - `Space` - select the current column; If the column is already selected, it will remove the selection;
+
+### Body navigation
+
+ When you focus the body of the grid and press one of the following key combinations, the described behavior is performed.
+
```

Comment:
The same applies to the **body**. Focusing that section reveals another bunch of combinations.